data/speakers/cercas.yml

```
@@ -0,0 +1,3 @@
+name: "Amanda Cercas Curry"
+image: "images/speakers/cercas.jpg"
+tagline: "Postdoctoral Researcher @ Bocconi"

```

```suggestion
tagline: "Postdoctoral Researcher @ Bocconi University"

```

magari non tutti la conoscono ;)